siriuspy/siriuspy/injctrl/csdev.py

```diff
@@ -417,6 +419,7 @@ def get_injctrl_propty_database():
             'type': 'char', 'count': 1000,
             'value': '\n'.join(_ct.INJ_STATUS_LABELS)},
     }
+    dbase = dbase.update(get_biasfb_database())
```

Comment:
bug: correct is just 

`dbase.update(get_biasfb_database())`

not 

`dbase = dbase.update(get_biasfb_database())`

Reply:
done